Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.42308&attempt_id=64f3f8e0-fed5-4275-a88a-4807b455728f)

Attempted: arc-icon-theme, faba-mono-icons, moka-icon-theme

<details><summary>Partial log (click to expand)</summary><p>

```
Installing subdir /build/source/Moka to /nix/store/kl5mf2ilvr7v0aagxmj4g9vj217ha6dr-moka-icon-theme-5.4.0/share/icons/Moka
Running custom install script '/build/source/meson/post_install.py'
Failed to run install script '/build/source/meson/post_install.py'
FAILED: meson-install
/nix/store/bypq8sfxwrzn5c9zpa03vif4vb8ly2kk-python3-3.6.5/bin/python3.6m /nix/store/456pjs8lg6ng0wyzsf0ksi5dd1j59c3c-meson-0.46.1/bin/meson --internal install /build/source/build/meson-private/install.dat
ninja: build stopped: subcommand failed.
builder for '/nix/store/0ba7hri4jn2v6q9r9qpm7kqcc5x4kb27-moka-icon-theme-5.4.0.drv' failed with exit code 1
cannot build derivation '/nix/store/7ja693g650a7wih96vvjadlxrcs6ibx4-arc-icon-theme-2016-11-22.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/98pskgr09zw2bclh8xa2fb6cak7j51an-faba-mono-icons-2016-04-30.drv': 1 dependencies couldn't be built
error: build of '/nix/store/0ba7hri4jn2v6q9r9qpm7kqcc5x4kb27-moka-icon-theme-5.4.0.drv', '/nix/store/7ja693g650a7wih96vvjadlxrcs6ibx4-arc-icon-theme-2016-11-22.drv', '/nix/store/98pskgr09zw2bclh8xa2fb6cak7j51an-faba-mono-icons-2016-04-30.drv' failed
```
</p></details>

--- a/pkgs/data/icons/arc-icon-theme/default.nix
+++ b/pkgs/data/icons/arc-icon-theme/default.nix
@@ -20,7 +20,8 @@ stdenv.mkDerivation rec {
     description = "Arc icon theme";
     homepage = https://github.com/horst3180/arc-icon-theme;
     license = licenses.gpl3;
-    platforms = platforms.all;
+    # moka-icon-theme dependency is restricted to linux
+    platforms = platforms.linux;
     maintainers = with maintainers; [ romildo ];
   };
 }

--- a/pkgs/data/icons/faba-icon-theme/default.nix
+++ b/pkgs/data/icons/faba-icon-theme/default.nix
@@ -1,24 +1,22 @@
-{ stdenv, fetchFromGitHub, autoreconfHook, elementary-icon-theme, gtk3 }:
+{ stdenv, fetchFromGitHub, meson, ninja, gtk3, elementary-icon-theme }:
 
 stdenv.mkDerivation rec {
   name = "${package-name}-${version}";
   package-name = "faba-icon-theme";
-  version = "2016-09-13";
+  version = "4.3";
 
   src = fetchFromGitHub {
     owner = "moka-project";
     repo = package-name;
-    rev = "00431894bce5fb1b8caccaee064788996be228a7";
-    sha256 = "0hif030pd4w3s851k0s65w0mf2pik10ha25ycpsv91gpbgarqcns";
+    rev = "v${version}";
+    sha256 = "0xh6ppr73p76z60ym49b4d0liwdc96w41cc5p07d48hxjsa6qd6n";
   };
 
-  nativeBuildInputs = [ autoreconfHook elementary-icon-theme gtk3 ];
+  nativeBuildInputs = [ meson ninja gtk3 elementary-icon-theme ];
 
   postPatch = ''
-    substituteInPlace Makefile.am --replace '$(DESTDIR)'/usr $out
+    patchShebangs meson/post_install.py
   '';
-
-  postFixup = "gtk-update-icon-cache $out/share/icons/Faba";
 
   meta = with stdenv.lib; {
     description = "A sexy and modern icon theme with Tango influences";

--- a/pkgs/data/icons/faba-mono-icons/default.nix
+++ b/pkgs/data/icons/faba-mono-icons/default.nix
@@ -24,7 +24,8 @@ stdenv.mkDerivation rec {
     description = "The full set of Faba monochrome panel icons";
     homepage = https://snwh.org/moka;
     license = licenses.gpl3;
-    platforms = platforms.all;
+    # moka-icon-theme dependency is restricted to linux
+    platforms = platforms.linux;
     maintainers = with maintainers; [ romildo ];
   };
 }

--- a/pkgs/data/icons/moka-icon-theme/default.nix
+++ b/pkgs/data/icons/moka-icon-theme/default.nix
@@ -1,24 +1,22 @@
-{ stdenv, fetchFromGitHub, autoreconfHook, faba-icon-theme, gtk3 }:
+{ stdenv, fetchFromGitHub, meson, ninja, gtk3, faba-icon-theme }:
 
 stdenv.mkDerivation rec {
   name = "${pname}-${version}";
   pname = "moka-icon-theme";
-  version = "5.3.6";
+  version = "5.4.0";
 
   src = fetchFromGitHub {
     owner = "snwh";
     repo = pname;
     rev = "v${version}";
-    sha256 = "17f8k8z8xvib4hkxq0cw9j7bhdpqpv5frrkyc4sbyildcbavzzbr";
+    sha256 = "015l02im4mha5z91dbchxf6xkp66d346bg3xskwg0rh3lglhjsrd";
   };
 
-  nativeBuildInputs = [ autoreconfHook faba-icon-theme gtk3 ];
+  nativeBuildInputs = [ meson ninja gtk3 faba-icon-theme ];
 
   postPatch = ''
-    substituteInPlace Makefile.am --replace '$(DESTDIR)'/usr $out
+    patchShebangs meson/post_install.py
   '';
-
-  postFixup = "gtk-update-icon-cache $out/share/icons/Moka";
 
   meta = with stdenv.lib; {
     description = "An icon theme designed with a minimal flat style using simple geometry and bright colours";

--- a/pkgs/data/icons/moka-icon-theme/default.nix
+++ b/pkgs/data/icons/moka-icon-theme/default.nix
@@ -22,7 +22,8 @@ stdenv.mkDerivation rec {
     description = "An icon theme designed with a minimal flat style using simple geometry and bright colours";
     homepage = https://snwh.org/moka;
     license = with licenses; [ cc-by-sa-40 gpl3 ];
-    platforms = platforms.all;
+    # darwin cannot deal with file names differing only in case
+    platforms = platforms.linux;
     maintainers = with maintainers; [ romildo ];
   };
 }

